functions/go/set-project-id/main.go

```diff
@@ -108,7 +108,7 @@ func setSettersOnKptfile(nodes []*yaml.RNode, kf *kptfilev1.KptFile, projectID s
 	}
 
 	fn := kptfilev1.Function{
-		Image: "gcr.io/kpt-fn/apply-setters:v0.1",
+		Image: "gcr.io/kpt-fn/apply-setters:v0.2",
```

Comment:
> I think it would be better for us(maintainers) to do that. Let's unblock this PR.

Can you please take care of it?

Reply:
(ended up taking Thursday/Friday off, so I didn't get a chance to work on this until today)

I updated my PR to incorporate the changes everywhere. [This](https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/9e34d68f4927b0be741535920856c588d0f786f2/functions/go/fix/fixpkg/fix.go#L284) is the only code change that isn't just a find/replace.

The E2E tests aren't passing, but they aren't passing on the base repo for me either, so perhaps I've configured them wrong.

Reply:
> > I think it would be better for us(maintainers) to do that. Let's unblock this PR.
> 
> Can you please take care of it?

Sorry about the confusion here. I was asking @phanimarupaka if he can help.